examples/custom_bindings/Cargo.toml

```diff
@@ -14,6 +14,7 @@ eframe = { version = "0.31.0", default-features = false, features = [
 
 log = "0.4"
 egui_term = { path = "../../" }
+wgpu = { workspace = true }
```

Comment:
It seems I got my branches mixed up. Here is the requested change for the other PR 🤣 

I'll get rid of this (in this branch only)

Reply:
done